pantry-finder-api/app/serializers/event_serializer.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# Defines Event attributes to be returned in JSON
+class EventSerializer < ActiveModel::Serializer
+  attributes :id, :address, :city, :state, :zip,
+             :pt_latitude, :pt_longitude, :loc_id
```

Comment:
You can remove `loc_id` from this, it's not used.

What do you think about aliasing pt_latititude and pt_longitude as just latitude and longitude? Might just be a quality of life improvement on the frontend.

Reply:
I think it could be useful to tie the agency id to the event through the loc_id.  It wouldn't be displayed to the user but it provides some insight into structural integrity and provide extensibility, i.e, at some point clients might want to access more details from the Location.

Reply:
The pt_latitude was more of a branding thing.  MOF has multiple geo coordinates for location.  pt_latitude was the one chosen.  I don't know if we would have multiple coordinates for different purposes.  I'd like to check with David on that.